frontend/src/pages/Extract/index.js

```diff
@@ -6,47 +6,79 @@ import './styles.css'
 import api from '../../service/api';
 import extractGif from '../../assets/extract.gif'
 import Hamburguer from '../../components/Hamburguer'
-import axios from 'axios';
 
 export default function Extract() {
         const [extract, setExtract] = useState([]);
         const [date, setDate] = useState('');
 
-        function HandleExtract(props){
+        function HandleExtractFirstTime(props){
             useEffect(()=>{
                 try {
-                    api.get('/transactions', { params: { user_id: localStorage.getItem('userId'), year_month: '2020-10' } }).then(res => {
-                        setExtract(res.data);
+                    var today = new Date();
+                    var currentDate;
+                    let currentMonth = today.getMonth()+1;
+                    let currentYear = today.getFullYear();
+
+                    currentDate = currentYear + '-' + currentMonth;
+   
+                    api.get('/transactions', { params: { user_id: localStorage.getItem('userId'), year_month: currentDate} }).then(res => {
+                        setExtract(res.data);                        
                     });
```

Comment:
Opcional: caso queira fazer com async/await

```suggestion
                  const response = await api.get('/transactions', { params: { user_id: localStorage.getItem('userId'), year_month: currentDate} })
                  setExtract(response.data);                        

```

Aí só teria que colocar o `async` antes da função

Reply:
Siim, eu tlgd mas acabei preferindo usar o .then